2024/README.md

```diff
@@ -24,7 +24,11 @@ Talks given and offers of talks for 2024. Please PR with your slides or talk off
  - Jost Berthold, A Tour of GHC's Garbage Collection [slides](./FPSyd202407_GHCGarbageCollection.pdf)
 
 ## August
- - ???
+ - Lightning talks
+   - Burin C, Closure-scripting all the things
```

Comment:
I managed to grab these references from Burin's talk, dumping here while they are fresh:
Clojure - Babashka and GraalVM
https://github.com/burinc/viip
https://github.com/hyperfiddle/electric
Jank Language
Hy Language
Rama Clojure

Reply:
@burinc is the title approved? I can add the links behind the title, or you can suggest changes here.

Reply:
Thanks @jberthold @tmcgilchrist for a great event.
If you you can add more links for those who may want to learn more about Clojure.

My talk title maybe `Automate all the things with Clojure/Babashka`

- [Babashka](https://babashka.org)
- [Deep Clojure/Python Integration](https://github.com/clj-python/libpython-clj)
- [python + clojure - examples](https://github.com/gigasquid/libpython-clj-examples)
- [Python + Clojure - OpenCV](https://github.com/gigasquid/libpython-clj-examples/tree/master/src/gigasquid/opencv)
- [https://hylang.org](https://hylang.org)
- [Jank](https://jank-lang.org)
- [Rama Clojure](https://blog.redplanetlabs.com/2023/10/11/introducing-ramas-clojure-api/)
- [viip - print source code with Clojure + GraalVM + Vim](https://github.com/burinc/viip)

Reply:
Thanks. Maybe the additional links can go to a small file of their own to keep this overview concise.
```suggestion
   - Burin C, Automate all the things with Clojure/[Babashka](https://babashka.org/). ([additional links](./closure-links-20240828.md))
```